summingbird-storm/src/main/scala/com/twitter/summingbird/storm/FlatMapBoltProvider.scala

```diff
@@ -79,10 +82,10 @@ case class FlatMapBoltProvider(builder: StormTopologyBuilder, node: StormNode) {
   private val maxEmitPerExecute = getOrElse(DEFAULT_MAX_EMIT_PER_EXECUTE)
   logger.info(s"[$nodeName] maxEmitPerExecute : ${maxEmitPerExecute.get}")
 
-  private val usePreferLocalDependency = getOrElse(DEFAULT_FM_PREFER_LOCAL_DEPENDENCY)
-  logger.info(s"[$nodeName] usePreferLocalDependency: ${usePreferLocalDependency.get}")
+  private val parallelism = getOrElse(DEFAULT_FM_PARALLELISM)
+  logger.info(s"[$nodeName] parallelism: ${parallelism.parHint}")
 
-  private def getFFMBolt[T, K, V](summer: SummerNode[Storm]) = {
+  private def getFFMBolt[T, K, V](summer: SummerNode[Storm]): Topology.Bolt[_, _] = {
```

Comment:
what are the `T, K, V` types here? I don't see why we need to pass type params that don't appear in the input or output, can we put them on the output?